pmm-app/src/pmm-check/components/AllChecksTab/AllChecksTab.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useEffect, useState } from 'react';
+import { cx } from 'emotion';
+import { CheckDetails } from 'pmm-check/types';
+import { CheckService } from 'pmm-check/Check.service';
+import { Spinner, useTheme } from '@grafana/ui';
+// TODO: make a table shared style
+import { getStyles as getTableStyles } from 'pmm-check/components/Table/Table.styles';
+import * as checkPanelStyles from 'pmm-check/CheckPanel.styles';
+import { Messages } from './AllChecksTab.messages';
+import * as styles from './AllChecksTab.styles';
+
+
+export const AllChecksTab: FC = () => {
+  const [fetchChecksPending, setFetchChecksPending] = useState(false);
+  const [checks, setChecks] = useState<CheckDetails[] | undefined>();
+  const theme = useTheme();
+  const tableStyles = getTableStyles(theme);
+
+  const fetchChecks = async (): Promise<void> => {
+    setFetchChecksPending(true);
+
+    try {
+      const checks = await CheckService.getAllChecks();
+
+      setChecks(checks);
+    } catch (err) {
+      console.error(err);
```

Comment:
Why don't we try the platform's logger to see how it works for PMM?
We plan on introducing an eslint rule against console, so maybe let's try.

Reply:
I will do that while working on SAAS-410, which will be merged on this one

Reply:
Ok, great!

Reply:
Since changing tests for this is not straightforward (let's discuss this separately) I'm going to open a ticket to use the logger everywhere in PMM

Reply:
Sounds great!